### ISSUE TITLE
Setup issue: Network owid-grapher_default not found

### ISSUE THREAD

**Issue by PremBharwani** [owid/owid-grapher#1358]
## Description

I am following the instructions mentioned [here](https://github.com/owid/owid-grapher/blob/master/docs/docker-compose-mysql.md). Towards the end while running the command `make up` I am running into an error as follows :

```
==> Checking your local environment has the necessary commands...
Copying .env.example-grapher --> .env
==> Downloading chart data
./devTools/docker/download-grapher-mysql.sh
Downloading live Grapher metadata database (owid_metadata)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 35.2M  100 35.2M    0     0  2298k      0  0:00:15  0:00:15 --:--:-- 2441k
Downloading live Grapher chartdata database (owid_chartdata)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  206M  100  206M    0     0   631k      0  0:05:35  0:05:35 --:--:-- 1883k
make[1]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
==> Validating your .env file for make up
make[2]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
make[2]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
.env file valid for make up
make[1]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
==> Building grapher
yarn install
yarn install v1.22.18
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 0.78s.
yarn run tsc -b
yarn run v1.22.18
$ /home/prembharwani/Desktop/owid_prem/owid-grapher/node_modules/.bin/tsc -b
Done in 0.28s.
==> Starting dev environment
tmux new-session -s grapher \
	-n docker 'docker-compose -f docker-compose.grapher.yml up' \; \
		set remain-on-exit on \; \
	new-window -n admin -e DEBUG='knex:query' \
		'DB_HOST=127.0.0.1 devTools/docker/wait-for-mysql.sh && yarn run tsc-watch -b --onSuccess "yarn startAdminServer"' \; \
		set remain-on-exit on \; \
	new-window -n webpack 'yarn run startSiteFront' \; \
		set remain-on-exit on \; \
	new-window -n welcome 'devTools/docker/banner.sh; exec  /usr/bin/zsh' \; \
	bind R respawn-pane -k \; \
	bind X kill-pane \; \
	bind Q kill-server \
	|| make down
tmux: unknown option -- e
no server running on /tmp/tmux-1000/default
make[1]: Entering directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'
==> Stopping services
docker-compose -f docker-compose.grapher.yml down
Removing network owid-grapher_default
WARNING: Network owid-grapher_default not found.
make[1]: Leaving directory '/home/prembharwani/Desktop/owid_prem/owid-grapher'

```

## Expected behaviour

As mentioned in the [setup guide](https://github.com/owid/owid-grapher/blob/master/docs/docker-compose-mysql.md), it should setup the database and the server should fire up and localhost:3030 should go live.

### Environment:
-   OS: Ubuntu 18.04

### Steps to reproduce:
1. Installing Docker 

    I installed the docker-engine and docker-compose from the docs ( References for what I followed :  [installing docker-engine](https://docs.docker.com/engine/install/ubuntu/) | [post install rootless setup](https://docs.docker.com/engine/install/linux-postinstall/)  | [installing docker-compose](https://docs.docker.com/compose/install/) )

2. Followed the instructions further from the [setup guide](https://github.com/owid/owid-grapher/blob/master/docs/docker-compose-mysql.md)

Basically after installing Docker, MySQL and nvm. I followed the below commands: 

```
 $ git clone https://github.com/owid/owid-grapher.git
 $ cd owid-grapher
 $ nvm install
 $ npm install -g yarn
 $ yarn
 $ yarn buildTsc
 $ yarn startStorybookServer
 $ cd .. && git clone https://github.com/owid/owid-content
 $ cd owid-grapher
 $ make up
```

Note that the local typescript development environment started off without errors and I could run the StoryBook server and I should be able to see the Database success message and localhost:3030 should respond.

### Tried workarounds:

Removing `-e` option from line 39 & 62, makes the command run and it opens up the `tmux` session. But that screen just freezes and I can't switch between panes. Also the server doesn't start at the port 3030.

**Comment by danyx23** [comment]
Hi @PremBharwani!

The cause might indeed be the -e flag with tmux. It seems that this needs tmux version 3.0 or higher. Can you rum `tmux -V` to verify which version of tmux you have and update to 3.0 or higher and let us know if that solves the problem?

**Comment by PremBharwani** [comment]
Hey @danyx23 ,

After the last commit made, and after upgrading the tmux version, it goes live! Thanks a lot!
Although I am still having difficulties connecting with the SQL DB, I am trying to figure it out myself. I would discuss regarding the same if I am unable to get it through. 

Thanks again :)
